### PR TITLE
Proofread docstrings through Factorize

### PR DESCRIPTION
**Pull request by mcneela**

### CODE REVIEW

File: composer/algorithms/factorize/factorize.py [hunk 1]
```diff
@@ -28,16 +28,16 @@ def apply_factorization(model: torch.nn.Module,
                         min_features: int = 512,
                         latent_features: Union[int, float] = 0.25,
                         optimizers: Optional[Union[Optimizer, Sequence[Optimizer]]] = None) -> torch.nn.Module:
-    """Replaces :class:`~torch.nn.Linear` and :class:`~torch.nn.Conv2d` modules and with
+    """Replaces :class:`~torch.nn.Linear` and :class:`~torch.nn.Conv2d` modules with
     :class:`~composer.algorithms.factorize.FactorizedLinear` and
     :class:`~composer.algorithms.factorize.FactorizedConv2d` modules.
 
-    Factorized modules replace one full-rank operation with a sequence of two
+    Factorized modules replace one, full-rank operation with a sequence of two,
```
Comment (dblalock) [inline composer/algorithms/factorize/factorize.py:35]: extra commas look weird to me; think these are cumulative adjectives that can't be reordered, and therefore don't get a comma: https://blogs.stockton.edu/helpwithpunctuation/files/2011/01/commas-with-adjectives.pdf

File: composer/algorithms/factorize/factorize.py [hunk 2]
```diff
@@ -47,8 +47,8 @@ def apply_factorization(model: torch.nn.Module,
             few channels are unlikely to be accelerated by factorization due
             to poor hardware utilization. Default: ``512``.
         latent_channels (int or float, optional): number of latent channels to use in factorized
-            convolutions. Can be specified as either an integer > 1 or as
-            float within [0, 1). In the latter case, the value is
+            convolutions. Can be specified as either an integer > 1 or as a
+            float within ``[0, 1)``. In the latter case, the value is
```
Comment (dblalock) [inline composer/algorithms/factorize/factorize.py:51]: for intervals, I'm pretty okay with anything as long as we're consistent, but I'm pretty sure I've seen torch write these as just raw text. Also as $$math$$, at least in [rand](https://pytorch.org/docs/stable/generated/torch.rand.html).

Just highlighting this as a thing we need to be careful to be consistent about; feel free to just resolve this comment.

File: composer/algorithms/factorize/factorize_core.py [hunk 3]
```diff
@@ -18,23 +18,26 @@ class LowRankSolution:
...
             the same size as the original operation.
-        bias: vector added to the output of the second linear operation
-        rank: output dimensionality (channels or features) of the first linear
-            operation, and input dimensionality of the second input operation.
-        nmse: normalized mean squared error obtained during the optimization
+        bias (:class:`torch.Tensor`, optional): vector added to the output of 
+            the second linear operation.
+        rank (int, optional): output dimensionality (channels or features) of the first linear
+            operation, and input dimensionality of the second input operation. Default: ``-1``.
+        nmse (float, optional): normalized mean squared error obtained during the optimization
```
Comment (dblalock) [inline composer/algorithms/factorize/factorize_core.py:34]: really nitpicky, but maybe shorter lines here for consistency with other docstrings?

**Review by dblalock — APPROVED** [review]
Awesome! Just a couple minor nitpicks.

File: composer/algorithms/factorize/factorize.py (hunk 1 above)
Comment (mcneela) [reply composer/algorithms/factorize/factorize.py:35]: You're right, my bad.

File: composer/algorithms/factorize/factorize_core.py (hunk 3 above)
Comment (mcneela) [reply composer/algorithms/factorize/factorize_core.py]: okay, fixed!

File: composer/algorithms/factorize/factorize.py (hunk 2 above)
Comment (mcneela) [reply composer/algorithms/factorize/factorize.py:51]: So should I change them to :math:? I've been consistently changing them to inline code, but I'm also fine with whatever.

**Comment by mcneela** [comment]
@hanlint I think this one is ready to merge as well. 🙂

**Pull request merged**